src/services/auth/oidc.ts

```diff
@@ -0,0 +1,60 @@
+import { HttpInstance, OidcService } from '../../types';
+import { HttpClient } from '../http-client';
+
+export default (
+  oidcClient: HttpClient,
+  httpWithAuth: HttpInstance
+): OidcService => ({
+  async createProvider(requestBody) {
+    const { data } = await oidcClient.post(
```

Comment:
The other endpoints don't extract `{ data }` they just return x.data. This is more of a preference for me and I can quickly change it back if we don't like it. Simple code yamsayin'.

Reply:
Yeah, this is way better 👍 I've a note for this as well. Idea right now was to refactor this away to be included in the `HttpClient` logic, as this is happening in basically every file 😬